.changeset/tsconfig-include-exclude.md

```diff
@@ -0,0 +1,22 @@
+---
+'sku': minor
+---
+
+Add new sku config options: [`tsconfigInclude`][tsconfiginclude] and [`tsconfigExclude`][tsconfigexclude], to control which files are included in the TypeScript compilation process. This is useful for excluding files that are not part of the TypeScript project, such as `node_modules` or `dist` folders.
+
+Previously, sku managed the `include` field in `tsconfig.json`, but this was problematic for projects that wanted more fine grained control over what is included and/or excluded from compilation. This became even more of a problem when tools like typescript-eslint would complain about some code not being included in the TypeScript project.
+
+> **Note**: If you were previously using [`srcPaths`][srcpaths] for this purpose, you should now use `tsconfigInclude` instead.
```

Comment:
https://github.com/orgs/community/discussions/16925

Reply:
I honestly don't like that entire note/warning syntax thing, for the same reasons other people don't like it. But it does stand out 🤷.